elements/pl-file-upload/pl-file-upload.js

```diff
@@ -34,16 +35,22 @@ window.PLFileUpload.prototype.initializeTemplate = function() {
         url: '/none',
         autoProcessQueue: false,
         accept: function(file, done) {
-            if (_.includes(that.acceptedFiles, file.name)) {
+            // fuzzy case:
+            const fileNameLowerCase = file.name.toLowerCase();
+            if (_.includes(that.acceptedFilesLowerCase, fileNameLowerCase)) {
                 return done();
             }
             return done('invalid file');
         },
         addedfile: function(file) {
-            if (!_.includes(that.acceptedFiles, file.name)) {
+            // fuzzy case match
+            const fileNameLowerCase = file.name.toLowerCase();
+            if (!_.includes(that.acceptedFilesLowerCase, fileNameLowerCase)) {
                 that.addWarningMessage('<strong>' + file.name + '</strong>' + ' did not match any accepted file for this question.');
                 return;
             }
```

Comment:
```suggestion
            }
            if (file.name !== fileNameLowerCase) {
                that.addWarningMessage(`<strong>${file.name}</strong> was automatically renamed to <strong>${fileNameLowerCase}</strong>.`);
            }
```
I know you explicitly said "rename silently" but maybe it'd be nice to show this message? Not sure, just an idea.

Reply:
We rename it to `acceptedName` when we save it, not the lowercase name. The lowercase version is only for finding the match.

Reply:
@tdy I'm not against the message but can we move it to target `acceptedName`?

Reply:
Sorry, just now my page refreshed. Heh.